capstone/src/main/webapp/index.html

```diff
@@ -31,13 +31,18 @@ <h2>Top Negative Associations:</h2>
             <li>Thing 3</li>
           </ul>
         </div>
-        <div class="stats-divider"></div>
         <div id="second-stats" class="container second-stats">
-          <p>Something here about stats</p>
+          <h2>Sentiment Percentages</h2>
+          <script src="https://www.gstatic.com/charts/loader.js"></script>
+          <script>
```

Comment:
Can this script be combined into a single JS function that is called? Nitpick but would be cleaner than having a two line script in the HTML IMO

Reply:
Just this one minor nitpick, other than that LGTM!

Reply:
I am not sure if that's possible, the charts API docs recommend placing the script tags in the html file

Reply:
Yeah its something worth looking into, personally I find it very messy to have js code in a script tag like that